Lib/typing.py

```diff
@@ -523,6 +524,28 @@ def returns_self(self) -> Self:
     raise TypeError(f"{self} is not subscriptable")
 
 
+@_SpecialForm
+def LiteralString(self, parameters):
+    """Represents an arbitrary literal string.
+
+    Example::
+
+        from typing import LiteralString
+
+        def query(sql: LiteralString) -> ...:
+            ...
+
+        query("SELECT * FROM table")  # ok
+        query(f"SELECT * FROM {input()}")  # not ok
```

Comment:
It may be worth including the `+` example here too. Otherwise, LGTM!

Reply:
Thanks, done